pom.xml

```diff
@@ -175,7 +175,7 @@
     <scala12.version>2.12.10</scala12.version>
     <scala.version>${scala12.version}</scala.version>
     <scala.collection-compat.version>2.8.1</scala.collection-compat.version>
-    <scala.binary.version>2.12</scala.binary.version>
+    <scala.binary.version>2.11</scala.binary.version>
```

Comment:
this is the default value which should be 2.12 because spark 3 is default now. If this is causing a problem, it means the test setup with spark 2.4 profile has some gap, which we need to only fix for that profile/setup

Reply:
Sorry! Forgot to remove this change. This was only for fixing the issues.